src/pages/trade/ui/summary.tsx

```diff
@@ -68,57 +60,82 @@ export const Summary = () => {
-            )}
-          >
-            <Text detail>{change24h.change}</Text>
+        {data && 'change' in data && (
+          <div className={cn('flex items-center gap-1', getColor(data, 'text'))}>
+            <Text detail>{roundTrim(data.change.value)}</Text>
             <span
-              className={cn(
-                'flex h-4 px-1 rounded-full text-success-dark',
-                change24h.positive ? 'bg-success-light' : 'bg-destructive-light',
-              )}
+              className={cn('flex h-4 px-1 rounded-full text-success-dark', getColor(data, 'bg'))}
             >
               <Text detail>
-                {change24h.positive ? '+' : '-'}
-                {change24h.percent}%
+                {getTextSign(data)}
+                {data.change.percent}%
               </Text>
             </span>
           </div>
         )}
       </SummaryCard>
       <SummaryCard title='24h High' loading={isLoading}>
-        {/*  TODO: After added to DB, show here */}
         <Text detail color='text.primary'>
-          -
+          {data && 'high' in data ? roundTrim(data.high) : '-'}
         </Text>
       </SummaryCard>
       <SummaryCard title='24h Low' loading={isLoading}>
-        {/*  TODO: After added to DB, show here */}
         <Text detail color='text.primary'>
-          -
+          {data && 'low' in data ? roundTrim(data.low) : '-'}
         </Text>
       </SummaryCard>
       <SummaryCard title='24h Volume' loading={isLoading}>
-        {data && (
-          <div className='flex items-center gap-1'>
-            {quoteAsset && <AssetIcon metadata={quoteAsset} size='sm' />}
-            <Text detail color='text.primary'>
-              {shortify(data.direct_volume_over_window)}
-            </Text>
-          </div>
+        {data && 'directVolume' in data ? (
+          <Density compact>
+            <ValueViewComponent
+              valueView={data.directVolume}
+              context='table'
+              abbreviate
+              hideSymbol
+            />
+          </Density>
+        ) : (
+          <Text detail color='text.primary'>
+            -
+          </Text>
         )}
       </SummaryCard>
     </div>
   );
 };
+
+// TODO: When rounding trim updated shipped in @penumbra-zone/types,
+//  can remove the removeTrailingZeros() wrapper
+const roundTrim = (value: number) => {
+  return removeTrailingZeros(round({ value, decimals: 6 }));
+};
+
+const getTextSign = (res: SummaryDataResponse) => {
+  if (res.change.sign === 'positive') {
+    return '+';
+  }
+  if (res.change.sign === 'negative') {
+    return '-';
+  }
+  return '';
+};
+
+const getColor = (res: SummaryDataResponse, version: 'text' | 'bg') => {
+  if (res.change.sign === 'positive') {
+    return `${version}-success-light`;
+  }
+  if (res.change.sign === 'negative') {
+    return `${version}-destructive-light`;
+  }
+  return `${version}-neutral-light`;
+};
```

Comment:
issue(not-so-blocking): this can only work if there's another component on the page with the same class defined statically. Would be better to extend the function to provide static classes to avoid the bug.

```suggestion
const getColor = (res: SummaryDataResponse, isBg = false): string => {
  if (res.change.sign === 'positive') {
    return isBg ? 'bg-success-light' : 'text-success-light';
  }
  if (res.change.sign === 'negative') {
    return isBg ? 'bg-destructive-light' : 'text-destructive-light';
  }
  return isBg ? 'bg-neutral-light' : 'text-neutral-light';
};
```

Reply:
Ah, TIL more about the tailwind compiler. Will update 👍